libff/algebra/fields/fp12_2over3over2.hpp

```diff
@@ -36,6 +36,14 @@ std::istream &operator>>(std::istream &, Fp12_2over3over2_model<n, modulus> &);
 template<mp_size_t n, const bigint<n> &modulus> class Fp12_2over3over2_model
 {
 public:
+    static bigint<12 * n> euler;            // (modulus^12-1)/2
```

Comment:
Let move the comments above the code lines to stop clang-format breaking the lines.

Reply:
Done